src/Database/PostgreSQL/Transact/Cluster/Connection.hs

```diff
@@ -18,6 +19,9 @@ module Database.PostgreSQL.Transact.Cluster.Connection (
     newClusterConnPool,
     newReadOnlyClusterConnPool,
     asReadOnlyPool,
+    connPool,
+    connPoolGeneric,
+    stripeActions,
```

Comment:
These don't necessarily need to be exported, but exporting them makes it easy to use them in GHCi to play around with.

This module isn't exposed, so it shouldn't be dangerous to export them.

Reply:
You can load the internal bindings of this module (or any module) in GHCi using 
```
:m +*Database.PostgreSQL.Transact.Cluster.Connection
```

I like the documentation and refactor but would really prefer to remove these bindings from the export list.

Reply:
@GambolingPangolin That sounds good.  I've removed them in 66963d9.